staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4168,6 +4168,15 @@ type PodStatus struct {
 	// +featureGate=InPlacePodVerticalScaling
 	// +optional
 	Resize PodResizeStatus `json:"resize,omitempty" protobuf:"bytes,14,opt,name=resize,casttype=PodResizeStatus"`
+
+	// TotalResourcesRequested is the effective resource requests for this pod taking into consideration init
+	// and regular containers, and the pod overhead. When the InPlacePodVerticalScaling feature gate is enabled,
+	// the field will show current effective resource request that is applied which may not match the requested value
```

Comment:
Does this semantic desirable? Will the semantic of what we want be better than what we have now?

Reply:
If it's "what we have", you can use it to determine the amount of free resources on a node where "what we want" isn't helpful for that. The name "TotalResourcesRequested" implies more of "what we want" though.

Reply:
After @vinaykul's comment, rebasing, renaming to `TotalRequestedResources` and changing it be "what we want" which more closely matches the name.